components/cyclebutton/index.css

```diff
@@ -11,9 +11,14 @@ governing permissions and limitations under the License.
 */
 
 .spectrum-CycleButton {
-  /* Be square */
-  padding-block: 0;
-  padding-inline: var(--spectrum-actionbutton-m-icononly-padding-left) var(--spectrum-actionbutton-m-icononly-padding-right);
+  /* 
+    force button to be square by negating
+    the :only-child selector from ActionButton-icon
+  */
+  &.spectrum-ActionButton-icon:not(:only-child) {
+    margin-inline-start: calc(var(--mod-actionbutton-edge-to-visual-only, var(--spectrum-actionbutton-edge-to-visual-only)) - var(--mod-actionbutton-edge-to-text, var(--spectrum-actionbutton-edge-to-text)));
+  }
```

Comment:
In place of the removed styles, I've had to override this `:only-child` rule that was coming from Action Button. This is a little duplicative, but I'm hesitant to change it inside of Action Button and it feels safer to just address it here.